dotnet/private/rules/nuget/nuget_archive.bzl

```diff
@@ -383,7 +383,7 @@ nuget_archive = repository_rule(
     _nuget_archive_impl,
     attrs = {
         "sources": attr.string_list(),
-        "netrc": attr.string(),
+        "netrc": attr.label(),
```

Comment:
Oh, wait, this change seems to be breaking absolute netrc path usage. I got err like this: `//external:nuget.system.text.json.v7.0.3: invalid label '/tmp/.netrc' in attribute 'netrc' in 'nuget_archive' rule: invalid target name '/tmp/.netrc': target names may not start with '/'`.

Reply:
And relative path does not seem to be working, because the default base dir is the cache dir for external pkg:
`ERROR: <my-bazel-cache-dir-for-external-pkg>/microsoft.netcore.app.runtime.linux-x64/7.0.8/BUILD.bazel:5:15: @mdmclient_deps//microsoft.netcore.app.runtime.linux-x64/7.0.8:7.0.8 depends on @nuget.microsoft.netcore.app.runtime.linux-x64.v7.0.8//:data in repository @nuget.microsoft.netcore.app.runtime.linux-x64.v7.0.8 which failed to fetch. no such package '@nuget.microsoft.netcore.app.runtime.linux-x64.v7.0.8//': Not a regular file: <my-bazel-cache-dir-for-external-pkg>/.netrc`

Reply:
It works with relative path by putting the netrc file under <WORKSPACE>/external/.

Reply:
Yes, this is a label to a file within the workspace. So e.g. if you have a `<WORKSPACE>/.netrc` you should put `//:netrc` into this attribute. I wasn't sure if I wanted to allow paths because relative paths won't work since the working directory is always under the output root and thus only absolute paths would work.

Reply:
Doesn´t the user .netrc fill the need for a global netrc? We default to it if there is not netrc declared in the attribute.

Reply:
The `read_user_netrc` method reads `${HOME}/.netrc`. In certain pipeline build scenario, the build tasks run in build agents. Bringing in user-relevant var like `${HOME}` is uncommon.
But I agree that using label to a file is more bazel nature comparing to a string. Probably I should just override `${HOME}` in my scenario.